### ISSUE TITLE
[Bug]: App does not build when using use_frameworks! in v11

### ISSUE THREAD

**Issue by balloman** [rnmapbox/maps#3137]
### Mapbox Implementation

Mapbox

### Mapbox Version

11.0.0-beta.5

### Platform

iOS

### `@rnmapbox/maps` version

10.1.0-beta.14

### Standalone component to reproduce

```javascript
import React from 'react';
import {
  MapView,
  ShapeSource,
  LineLayer,
  Camera,
} from '@rnmapbox/maps';

const aLine = {
  type: 'LineString',
  coordinates: [
    [-74.00597, 40.71427],
    [-74.00697, 40.71527],
  ],
};

class BugReportExample extends React.Component {
  render() {
    return (
      <MapView style={{flex: 1}}>
        <Camera centerCoordinate={[-74.00597, 40.71427]} zoomLevel={14} />
        <ShapeSource id="idStreetLayer" shape={aLine}>
          <LineLayer id="idStreetLayer" />
        </ShapeSource>
      </MapView>
    );
  }
}
```

### Observed behavior and steps to reproduce

When building the application on iOS, the error 
```

  15 | @end
  16 | 
> 17 | #import <rnmapbox_maps-Swift.h>
     |         ^ 'rnmapbox_maps-Swift.h' file not found
  18 | 
  19 | 
  20 | @implementation RNMBXShapeSourceModule
```
is generated by RNMBShapeSourceModule.mm

### Expected behavior

There is no error.

### Notes / preliminary analysis

This can be fixed by using the same method that was applied in rnmapbox_maps-Swift.pre.h, so modifying the import to check if use frameworks is being used like so: 
```objc
#if RNMBX_USE_FRAMEWORKS
#import <rnmapbox_maps/rnmapbox_maps-Swift.h>
#else
#import <rnmapbox_maps-Swift.h>
#endif
```

**Comment by mfazekas** [comment]
@balloman thanks yep, there were new #import <rnmapbox_maps ... added those should be replace with `#import "rnmapbox_maps-Swift.pre.h"`

**Comment by mfazekas** [comment]
Should be fixed by: #3138

**Comment by herbertvuijk** [comment]
Add 
$RNMapboxMapsUseFrameworks = true
to your POD file if you use_frameworks, just below:
$RNMapboxMapsImpl = 'mapbox'

Then run pod install and try again.
